hooks/path_info.py

```diff
@@ -706,6 +706,55 @@ def copy_files(self, src_files, dest_path, seal_files=False, is_sequence=False):
 
         return processed_files
 
+    def copy_folder(self, src_folders, dest_folder, seal_folder=False):
```

Comment:
It doesn't make sense for this to be a list of src_folders. `filesystem.copy_folder()` replace copies the exact folder to the dst path, which means if we had a list, we would be replacing the dir each time.
A list is accepted in `copy_files()`, since we anticipate a sequence of files (i.e. like frames) to be copied to a dst path with an expandable token such as `%04d`.
If we anticipate folders with seq token, please handle the `is_sequence` property. Else, please make this accept a single src folder and update the calling methods.